apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/db/model/VersionedDeletableEntity.java

```diff
@@ -0,0 +1,26 @@
+package com.apollocurrency.aplwallet.apl.core.db.model;
+
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.Setter;
+import lombok.ToString;
+
+import java.sql.ResultSet;
+import java.sql.SQLException;
+@EqualsAndHashCode(callSuper = true)
+@ToString(callSuper = true)
+public class VersionedDeletableEntity extends VersionedDerivedEntity {
+    @Getter
+    @Setter
+    private boolean deleted = false;
```

Comment:
it's false by default (see https://docs.oracle.com/javase/tutorial/java/nutsandbolts/datatypes.html, Default Values)

Reply:
done